pkpdapp/pkpdapp/migrations/0005_auto_20210318_1408.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.0.7 on 2021-03-18 14:08
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
```

Comment:
please have all schema changes merged into the initial 0001 migration

Reply:
Done